go.mod

```diff
@@ -4,7 +4,7 @@ go 1.22
 
 require (
 	github.com/IBM/sarama v1.41.1
-	github.com/cloudtrust/common-service/v2 v2.7.0
+	github.com/cloudtrust/common-service/v2 v2.7.1
```

Comment:
Looks like last version is [v2.8.1](https://github.com/cloudtrust/common-service/releases/tag/v2.8.1)

Reply:
Yes but this release updates dependencies and we do not want to so it yet